recipes/fortran-magic/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7" %}
+
+package:
+    name: fortran-magic
+    version: {{ version }}
+
+source:
+    fn: fortran-magic-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/f/fortran-magic/fortran-magic-{{ version }}.tar.gz
+    md5: dc57d9c5daa0e16f6ce315045481b63e
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - ipython
```

Comment:
Actually, this might not be required at all.

I believe we are just dropping some kernel spec files into a location for Jupyter client to find them. Though, I think there is another install step required. See this [example]( http://ipython.readthedocs.org/en/stable/install/kernel_install.html#kernels-for-python-2-and-3 ) with `ipykernel`.

At most, we might need `jupyter_client`, but I'm not sure that it is even necessary to install the kernel. Still users might expect something they can install and do something with. So we need to find a balance between an install with the required dependencies and a usable install.

Here is Continuum's [`irkernel` recipe]( https://github.com/conda/conda-recipes/blob/master/r-packages/r-irkernel/meta.yaml ). Note it does not install any Jupyter packages.